### PR TITLE
Feat: Reordering documentation to add more clairty [APE-1822]

### PR DESCRIPTION
**Pull request by rafael-abuawad**
### What I did
I moved the **Fixtures** section on top of the **Test Pattern** section as mentioned on this issue [APE-1822](https://github.com/ApeWorX/ape/issues/1822)

### Checklist
- [ ] Is consistent
- [ ] Is good documentation
- [ ] Documentation has been updated

### CODE REVIEW

File: docs/userguides/testing.md [hunk 1]
```diff
@@ -187,6 +155,40 @@ def my_contract(Contract):
...
+    with ape.reverts("!authorized"):
+        my_contract.authorized_method(sender=not_owner)
+```
+
+```{note}
+Ape has built-in test and fixture isolation for all pytest scopes.
+To disable isolation add the `--disable-isolation` flag when running `ape test`
+```
+
+
```
Comment (antazoey) [inline docs/userguides/testing.md:190]: too many lines

File: docs/userguides/testing.md [hunk 2]
```diff
@@ -187,6 +155,40 @@ def my_contract(Contract):
 
 It has the same interface as the [ChainManager](../methoddocs/managers.html#ape.managers.chain.ChainManager).
 
+## Test Pattern
```
Comment (antazoey) [inline docs/userguides/testing.md:158]: should rename this to like `"Example"` or something, as the intent has changed by moving it away from the top of the article. It was supposed to be like "this is what high-level tests should be", but when moved down here, it is more like "this is a complete example".

**Review by antazoey — CHANGES_REQUESTED** [review]
ideas

**Comment by antazoey** [comment]
can we do this instead? https://github.com/ApeWorX/ape/pull/1829/files

**Comment by fubuloubu** [comment]
Thank you @rafael-abuawad for contributing!

I think per our discussion in #1822, that #1829 is a better fix

closing in favor of #1829

**Pull request closed without merging**